dask/array/core.py

```diff
@@ -1270,6 +1270,29 @@ def coarsen(reduction, x, axes):
     return Array(merge(x.dask, dsk), name, blockdims=blockdims, dtype=dt)
 
 
+@wraps(chunk.broadcast_to)
+def broadcast_to(x, shape):
+    shape = tuple(shape)
+    ndim_new = len(shape) - x.ndim
+    if ndim_new < 0 or any(new != old
+                           for new, old in zip(shape[ndim_new:], x.shape)
+                           if old != 1):
+        raise ValueError('cannot broadcast shape %s to shape %s'
+                         % (x.shape, shape))
+
+    name = next(names)
+    blockdims = (tuple((s,) for s in shape[:ndim_new])
```

Comment:
Do we always want solid, un-cut blocks in the new dimensions?  Or maybe this is a case where we reblock afterwards.